model/system_parameters.py

```diff
@@ -206,7 +207,8 @@ class Parameters:
 
     validator_process: List[Callable[[Run, Timestep], int]] = default(
         [
-            lambda _run, _timestep: 3,
+            # From https://beaconscan.com/statistics as of 20/04/21
+            lambda _run, _timestep: subgraph.get_6_month_mean_validator_deposits_per_epoch(default=3),
```

Comment:
There's a section at the top of this file where API calls are made and assigned to variables, let's do that and assign variable "mean_validator_deposits_per_epoch".

This doesn't look like it's up to date with main - specifically `# From https://beaconscan.com/statistics as of 20/04/21`. And also docstring needs to be updated. We can use the existing descriptions to tell people where the values 3 comes from.